nilearn/masking.py

```diff
@@ -656,7 +673,73 @@ def compute_multi_gray_matter_mask(target_imgs, threshold=.5,
 
     threshold: float, optional
         The value under which the MNI template is cut off.
-        Default value is 0.5
+        Default value is 0.5.
+
+    connected: bool, optional
+        if connected is True, only the largest connect component is kept.
+        Default is True
+
+    opening: bool or int, optional
+        if opening is True, a morphological opening is performed, to keep
+        only large structures.
+        If opening is an integer `n`, it is performed via `n` erosions.
+        After estimation of the largest connected constituent, 2`n` closing
+        operations are performed followed by `n` erosions. This corresponds
+        to 1 opening operation of order `n` followed by a closing operator
+        of order `n`.
+
+    memory: instance of joblib.Memory or str
+        Used to cache the function call.
+
+    n_jobs: integer, optional
+        Argument not used but kept to fit the API
+
+    **kwargs: optional arguments
+        arguments such as 'target_affine' are used in the call of other
+        masking strategies, which then would raise an error for this function
+        which does not need such arguments.
+
+    verbose: int, optional
+        Controls the amount of verbosity: higher numbers give
+        more messages
+
+    Returns
+    -------
+    mask: nibabel.Nifti1Image
+        The brain mask (3D image)
+
+    See also
+    --------
+    nilearn.masking.compute_brain_mask
+    """
+    return compute_multi_brain_mask(target_imgs=target_imgs,
+                                    threshold=threshold, connected=connected,
+                                    opening=opening, memory=memory,
+                                    verbose=verbose, n_jobs=n_jobs,
+                                    mask='whole-brain', **kwargs)
+
+
+def compute_multi_brain_mask(target_imgs, threshold=.5, connected=True,
+                             opening=2, memory=None, verbose=0, n_jobs=1,
+                             mask='whole-brain', **kwargs):
```

Comment:
why is it called "mask" here and "mask_type" in `compute_brain_mask`? unless there is a reason that I missed, could you use "mask_type" for both?

Reply:
No reason at all. I just changed it.